packages/deployment/app/cloudformation.template.yml

```diff
@@ -777,6 +777,7 @@ Resources:
           Fn::Join:
             - ""
             - - "#cloud-boothook\n"
+              - "mkdir -p /etc/docker && echo '{\n  \"storage-driver\": \"devicemapper\"\n}' >> /etc/docker/daemon.json\n"
```

Comment:
what does this do?

Reply:
This enables the `devicemapper` storage driver https://docs.docker.com/storage/storagedriver/device-mapper-driver/